src/3rdParty/salomesmesh/src/SMDS/SMDS_UnstructuredGrid.cpp

```diff
@@ -1026,7 +1026,12 @@ void SMDS_UnstructuredGrid::BuildLinks()
   GetLinks()->Allocate(this->GetNumberOfPoints());
   GetLinks()->Register(this);
 //FIXME: vtk9
+  #if VTK_VERSION_NUMBER < VTK_VERSION_CHECK(9,3,0)
   GetLinks()->BuildLinks(this);
+  #else
+  GetLinks()->SetDataSet(this);
+  GetLinks()->BuildLinks();
+  #endif
```

Comment:
how about:
```suggestion
  #if VTK_MAJOR_VERSION >= 9 && VTK_MINOR_VERSION >= 3
  GetLinks()->SetDataSet(this);
  GetLinks()->BuildLinks();
  #else
  GetLinks()->BuildLinks(this);
  #endif
```
these should be available on 7.1: https://gitlab.kitware.com/vtk/vtk/-/blob/v7.1.0/Common/Core/vtkVersionMacros.h.in?ref_type=tags